tests/pages/components/task-comment-list.js

```diff
@@ -0,0 +1,11 @@
+import {
+  collection
+} from 'ember-cli-page-object';
+export default ({
```

Comment:
You should add a line between this `export`, and remove the parens wrapping it. Much prefer the `export default {` style.